tutorials/11-probabilistic-pca/11_probabilistic-pca.jmd

```diff
@@ -63,82 +79,72 @@ We use PCA to explore underlying structure or pattern which may not necessarily
 
 #### Step 1: configuration of dependencies
 
-First, we define the dependencies used.
+First, we load the dependencies used.
 
 ```julia
+using Pkg: Pkg;
+Pkg.add("FillArrays");
+using Pkg: Pkg;
+Pkg.add("ReverseDiff");
```

Comment:
These should be managed by `Project.toml` and `Manifesto.jl`. See https://pkgdocs.julialang.org/v1/environments/ for a good introduction. 

```suggestion
```

Reply:
@yebai Thanks for pointing me to the guide. I was about to ask if changes can be made in `Project.toml` when adding them/ Just a quick test to see if the building is successful before modifying the project file.